contracts/BaseWorkflow.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.23;
+
+import { IAccessController } from "@storyprotocol/core/interfaces/access/IAccessController.sol";
+import { IIPAssetRegistry } from "@storyprotocol/core/interfaces/registries/IIPAssetRegistry.sol";
+import { ILicenseRegistry } from "@storyprotocol/core/interfaces/registries/ILicenseRegistry.sol";
+import { ILicensingModule } from "@storyprotocol/core/interfaces/modules/licensing/ILicensingModule.sol";
+import { ICoreMetadataModule } from "@storyprotocol/core/interfaces/modules/metadata/ICoreMetadataModule.sol";
+import { IPILicenseTemplate } from "@storyprotocol/core/interfaces/modules/licensing/IPILicenseTemplate.sol";
+
+import { ISPGNFT } from "./interfaces/ISPGNFT.sol";
+import { SPGNFTLib } from "./lib/SPGNFTLib.sol";
+import { Errors } from "./lib/Errors.sol";
+
+/// @title Base Workflow
+/// @notice The base contract for all Story Protocol Periphery workflows.
+contract BaseWorkflow {
```

Comment:
Can be marked at abstract contract?

Reply:
Resolved in [`28f046e`](https://github.com/storyprotocol/protocol-periphery-v1/pull/43/commits/28f046ee2434b61c4aae5a318f0d1c4bb2b4ee5a) :)